admin_guide/manage_rbac.adoc

```diff
@@ -762,8 +762,158 @@ To bind the new role to a user, run:
+See custom columns [http://kubernetes.io/docs/user-guide/kubectl-overview/#custom-columns], golang template
+[http://golang.org/pkg/text/template/#pkg-overview] and jsonpath template
+[http://kubernetes.io/docs/user-guide/jsonpath].
+      --resource=[]: Resource that the rule applies to
+      --resource-name=[]: Resource in the white list that the rule applies to, repeat this flag for multiple items
+      --save-config=false: If true, the configuration of current object will be saved in its annotation. Otherwise, the
+annotation will be unchanged. This flag is useful when you want to perform kubectl apply on this object in the future.
+      --template='': Template string or path to template file to use when -o=go-template, -o=go-template-file. The
+template format is golang templates [http://golang.org/pkg/text/template/#pkg-overview].
+      --validate=false: If true, use a schema to validate the input before sending it
+      --verb=[]: Verb that applies to the resources contained in the rule
+
+----
+
+[[creating-derivative-cluster-role]]
+== Creating a cluster role from another cluster role
+
+. Export a cluster role to modify:
++
+----
+oc get clusterrole admin -o yaml > newrole.yaml
+----
++
+You can export any cluster role, including a
+xref:../architecture/additional_concepts/authorization.adoc#roles[default cluster role],
+to start your new role.
+
+. Customize the role definition in the *_newrole.yaml_* file.
++
+----
+aggregationRule: <1>
+  clusterRoleSelectors:
+  - matchLabels:
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+apiVersion: authorization.openshift.io/v1
+kind: ClusterRole
+metadata: 
+  annotations: 
+    openshift.io/description: "A user that has rights." <2>
+    openshift.io/reconcile-protect: "false"
+  creationTimestamp: 2018-04-06T15:04:05Z
+  labels:
+	  kubernetes.io/bootstrapping: rbac-defaults
+  name: <new_cluster_role> <3>
+  resourceVersion: "2321"
+  selfLink: /apis/authorization.openshift.io/v1/clusterroles/<new_cluster_role> <3>
+  uid: bfbf991e-39ab-11e8-bb7b-080027a54c8e
+rules: <4>
+- apiGroups:
+  - ""
+  attributeRestrictions: null
+  resources:
+  - pods
+  - pods/attach
+  - pods/exec
+  - pods/portforward
+  - pods/proxy
+  verbs:
+  - create
+  - delete
+  - deletecollection
+  - get
+  - list
+  - patch
+  - update
+  - watch
+...[OUTPUT OMITTED]...
+- apiGroups:
+  - settings.k8s.io
+  attributeRestrictions: null
+  resources:
+  - podpresets
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - update
+  - watch
+----
+<1> Remove the `aggregationRule` section.
+<2> Optional: Update the role description.
+<3> Provide the name for the new cluster role.
+<4> Customize the cluster role rules as required.
+
+. Create the role:
++
+----
+$ oc create -f newrole.yaml
+----
+
+////
+. Grant the permissions in the cluster role to a user:
+.. To grant a user permissions in all projects:
++
+----
+$ oc adm policy add-cluster-role-to-user <new_cluster_role> <username>
+----
+
```

Comment:
Maybe we should remind that there's also another way to bind a {cluster-}role to users. `oc create {cluster}role{binding}`

Reply:
Is there a different time that a user would want to do this? @enj, what do you think?

Reply:
@enj, when you have a minute, will you PTAL?

Reply:
After a chat with @enj, I think that `add-cluster-role-to-user` is the right way to go.